.github/workflows/master-pr-build.yml

```diff
@@ -71,14 +71,14 @@ jobs:
           distribution: 'temurin'
           java-version: ${{ matrix.java }}
           cache: 'maven'
-      - name: maven build
-        run: ${{ steps.install-mvnd.outputs.mvnd-dir }}/mvnd -l build.log -Dmvnd.threads=2 -V -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 --no-transfer-progress -Pfastinstall -Dcheckstyle.skip=true -DskipTests install
-      - name: archive logs
-        uses: actions/upload-artifact@v3
-        if: always()
-        with:
-          name: build.log
-          path: build.log
+#      - name: maven build
+#        run: ${{ steps.install-mvnd.outputs.mvnd-dir }}/mvnd -l build.log -Dmvnd.threads=2 -V -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 --no-transfer-progress -Pfastinstall -Dcheckstyle.skip=true -DskipTests install
+#      - name: archive logs
+#        uses: actions/upload-artifact@v3
+#        if: always()
+#        with:
+#          name: build.log
+#          path: build.log
```

Comment:
This is preventing the build logs from being uploaded.

Reply:
I know, it is only to get a quick feedback (avoid waiting for the maven build) as the commit message states. I will of course remove this commit once the quick test will pass, sorry for the noise

Reply:
No worries, thanks for the heads up.